docs/users_and_security/user_meta.md

```diff
@@ -142,5 +143,7 @@ use CodeIgniter\Entity;
 class CustomEntity extends Entity
 {
     use HasMeta;
+
+    public string $configClass = 'CustomConfig';
```

Comment:
This isn't needed outside of the class or the trait, so let's call it protected. 
```suggestion
    protected string $configClass = 'CustomConfig';
```

Reply:
Implemented this change